wordstats/cognate_db.py

```diff
@@ -0,0 +1,119 @@
+from sys import stdout
+
+import sqlalchemy.orm
+import sqlalchemy
+from sqlalchemy import Column, Integer, String, UniqueConstraint, Boolean
+
+from .base_service import Base, SimplifiedQuery
+
+
+class UnknownCognateInfo(object):
+    """ Default values if  we have no information about
+    a given cognate pair """
+    def __init__(self):
+        self.word_from = ""
+        self.word_to = ""
+        self.blacklist = False
+        self.whitelist = False
+
+
+# structure for cognate candidates not reviewed by the user
+class CognateCandidatesInfo(SimplifiedQuery, Base):
+    __tablename__ = 'candidates_info'
+    __table_args__ = {'mysql_collate': 'utf8_bin'}
+
+    id = Column(Integer, primary_key=True)
+    method = Column(String(255), nullable=False, index=True)
+
+    word_from = Column(String(255), nullable =False, index = True)
+    word_to = Column(String(255), nullable=False, index=True)
+    primary = Column(String(20), nullable =False, index = True)
+    secondary = Column(String(20), nullable=False, index=True)
+
+    UniqueConstraint(word_from, word_to, primary, secondary, method)
+
+    def __init__(self, word_from, word_to, languageFrom, languageTo, method):
+        self.word_from = word_from
+        self.word_to = word_to
+        self.primary = languageFrom
+        self.secondary = languageTo
```

Comment:
i think we should keep a parallel structure between the word and the language. 
thus, either: 
- word_from  / language_from
- or word_primary / language_primary